crates/bitcoin-da/src/service.rs

```diff
@@ -146,13 +145,27 @@ impl BitcoinService {
                             }
                         };
                         match self
-                            .send_transaction_with_fee_rate(prev.clone(), blob, fee_sat_per_vbyte)
+                            .send_transaction_with_fee_rate(
+                                prev.clone(),
+                                &request.da_data,
+                                fee_sat_per_vbyte,
+                            )
                             .await
                         {
                             Ok(tx) => {
                                 let tx_id = TxidWrapper(tx.id);
                                 info!(%tx.id, "Sent tx to BitcoinDA");
-                                prev_tx = Some(tx);
+                                prev_utxo = Some(UTXO {
+                                    tx_id: tx.id,
+                                    vout: 0,
+                                    script_pubkey: tx.tx.output[0].script_pubkey.to_hex_string(),
+                                    address: None,
```

Comment:
@ercecan you set address to Some("ANY"). Can I use None here as well?

Reply:
I believe it is okay yes